hoomd/typeconverter.py

```diff
@@ -261,30 +248,7 @@ def toTypeConverter(value):
         return TypeConverterFixedLengthSequence(value)
```

Comment:
Shouldn't this be named `to_type_converter`?

Reply:
Yes, I was just using it as a "pretend" class. Done.